tests/test_pdf.py

```diff
@@ -372,6 +372,4 @@ def test_invalid_modifier_name_resuse():
         ]
     }
     with pytest.raises(pyhf.exceptions.InvalidNameReuse):
-        pdf = pyhf.Model(spec, poiname='reused_name')
-
-    pdf = pyhf.Model(spec, poiname='reused_name', qualify_names=True)
```

Comment:
Please correct me if I'm missing a way that this should be changed and not removed.